filament/backend/src/vulkan/VulkanPipelineCache.h

```diff
@@ -298,11 +326,9 @@ class VulkanPipelineCache : public CommandBufferObserver {
     utils::bitset32 mDirtyPipeline;
     utils::bitset32 mDirtyDescriptor;
 
-    VkDescriptorSetLayout mDescriptorSetLayouts[DESCRIPTOR_TYPE_COUNT] = {};
-
-    std::vector<VkDescriptorSet> mDescriptorSetArena[DESCRIPTOR_TYPE_COUNT];
+    tsl::robin_map<VkPipelineLayout, VkDescriptorSet> mDescriptorSetArena[DESCRIPTOR_TYPE_COUNT];
```

Comment:
I wonder if these arenas could be moved into LayoutBundle.  If they lived there, then they could be stored in a simple vector rather than a map.  What do you think?

Reply:
That's good point. It's changed.